src/sc_generation/template/env_wasmv1.ts

```diff
@@ -0,0 +1,1479 @@
+import * as proto from "../../../../massa-abi-metaproject/massa-proto-as/assembly";
```

Comment:
we can't merge this ^^

I think we should generate this file by fetching it from as_abi_protobuf/assembly/env.ts

Note1: as_abi_protobuf is a temporary repository, still some work here (merge in massa-as-sdk ?)

Note2: if we generate this file, we should also check automatically the list against what's inside massa-sc-runtime abis.rs

Reply:
Should I set massa-proto-as as a dependancy in the template/package.json and document that you should npm install?
I will add it to the template so it's here even without npm install, but for syncing with the project?

Reply:
I'm not sure if it's immediate to use as_abi_protobuf/assembly/env.ts directly, because there are some changes (env.make_* for more than amounts, etc), and the formatting of this file is important for the parser...